src/main/resources/schema/wildfly-elytron_1_0.xsd

```diff
@@ -1569,6 +1570,52 @@
         </xs:complexContent>
     </xs:complexType>
 
+    <xs:complexType name="constantPermissionMapperType">
+        <xs:annotation>
+            <xs:documentation>
+                A RoleMapper definition that always returns a pre-defined set of permissions.
```

Comment:
We may have other typos in this area - should be PermissionMapper not RoleMapper.

Once this PR is merged could probably use a follow up just to correct this.